src/components/common/button/Chip.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import classNames from 'classnames';
+
+/**
+ * chip 컴포넌트
+ * @component
+ *
+ * @param {('day' | 'default')} [variant='default'] - 칩의 변형
+ * @param {boolean} selected - 칩의 선택 상태
+ *
+ * const [isSelectedLarge, setIsSelectedLarge] = useState(false);
+ * <Chip size="lg" selected={isSelectedLarge} onClick={() => void}>
+ *    Day 1
+ * </Chip>
+ */
+
+const chipStyles: { [key: string]: string } = {
+  base: 'flex px-5 py-1.75 justify-center items-center gap-2.5 rounded-full font-caption-1',
+  selectedDay: 'bg-black-01 text-white-01',
+  selectedDefault: 'border-solid border-secondary-01 bg-white text-secondary-01',
+  unselectedDay: 'border-solid border-gray-02 bg-white-01 text-black-02',
+  unselectedDefault: 'border-solid border-gray-02 bg-white-01 text-black-02',
+  day: 'h-10',
+  default: 'h-9'
+};
```

Comment:
```suggestion
const chipStyles: { [key: string]: string | {day: string, default: string} } = {
  base: 'flex px-5 py-1.75 justify-center items-center gap-2.5 rounded-full font-caption-1',
  selected: {
    day: 'bg-black-01 text-white-01',
    default: 'border-solid border-secondary-01 bg-white text-secondary-01'
  },
  unselected: {
    day: 'border-solid border-gray-02 bg-white-01 text-black-02',
    default:  'border-solid border-gray-02 bg-white-01 text-black-02'
  },
  day: 'h-10',
  default: 'h-9'
};
```

밑에 37줄부터 43줄까지 삼항연산자의 깊이가 생겨서 객체로 분리하는 방법도 생각해봤습니다~
근데 타입이 복잡해지는 거 같아서 더 나은지는 모르겠네요

Reply:
나중에 확장성을 생각하면 좋은 방법같아요! 반영했습니다👍